<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addSignalDialog</class>
 <widget class="QDialog" name="addSignalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add tag</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Add tag</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="edit_IOA"/>
      </item>
      <item row="3" column="1">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Single point information (M_SP_NA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Double point information (M_DP_NA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Step position information (M_ST_NA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bit string of 32 bit (M_BO_NA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Measured value, normalized value (M_ME_NA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Measured value, scaled value (M_ME_NB)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Measured value, floating point (M_ME_NC)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Integrated totals (M_IT_NA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Packed single-point information (M_PS_NA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Measured value, normalized value without quality descriptor (M_ME_ND)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Single point information with time tag (M_SP_TB)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Double point information with time tag  (M_DP_TB)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Step position with time tag (M_ST_TB)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bit string of 32 bit with time tag (M_BO_TB)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Measured value, normalized value with time tag (M_ME_TD)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Measured value, scaled value with time tag (M_ME_TE)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Measured value, floating point with time tag (M_ME_TF)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Integrated totals with time tag (M_IT_TB)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Event of protection equipment with time tag (M_EP_TD)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Packed start events of protection equipment with time tag (M_EP_TE)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Packed output circuit information of protection equipment with time tag (M_EP_TF)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_ioa">
        <property name="text">
         <string>IOA</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_type">
        <property name="text">
         <string>type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_descr"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_desc">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addSignalDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addSignalDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
